plugins/baseball-scoreboard/base_odds_manager.py

```
@@ -0,0 +1,293 @@
+            league: League name (e.g., 'nfl', 'nba')
+            event_id: ESPN event ID
+            update_interval_seconds: Override default update interval
+
+        Returns:
+            Dictionary containing odds data or None if unavailable
+        """
+        if sport is None or league is None:
+            raise ValueError("Sport and League cannot be None")
+
+        # Use provided interval or default
+        interval = update_interval_seconds or self.update_interval
```

Ruff (0.15.0)

[error] Local variable `interval` is assigned to but never used

Remove assignment to unused variable `interval`

(F841)